### DISCUSSION TITLE
combining telemetry point

### DISCUSSION THREAD

**Discussion by alexhoffenreich** [nasa/openmct#7460] [Community Q&A] [answered]
Is there a plug-in that I could combine values of different telemetry points into a new point?
I.e.: 
I drag two telemetry points to the plug-in, then write a function in javascript in the form that combines between them, like:
p => Math.sqrt(p[0] **2 + p[1]**2)
or
p=> p.map(p[0] %2 ==0? "even": "odd")

and the new telemetry point represents the result

If not exactly, is there anything that can provide similar functionality?

**Comment by davetsay** [comment, marked as answer]
@alexhoffenreich , you'll be happy to know that there is an issue #7823, and an open PR, which will add this functionality to Open MCT
